src/anbox/cmds/container_manager.cpp

```diff
@@ -51,6 +51,13 @@ anbox::cmds::ContainerManager::ContainerManager()
 
   action([&](const cli::Command::Context&) {
     try {
+      if (getuid() != 0) {
+        ERROR("You're not running the container-manager as root. Generally you don't");
```

Comment:
I did this before but that breaks the formatting as we have then

ERROR [test.cpp:100] You're not running the container-manager as root. Generally you don't
want to run the container-manager manually unless you're a developer
as it is started by the init system of your operating system.